src/br/ita/toner/answer/BitSetAnswer.java

```diff
@@ -0,0 +1,33 @@
+package br.ita.toner.answer;
+import java.util.BitSet;
+import java.util.List;
+import java.util.Vector;
+
+public class BitSetAnswer implements Answer<List<BitSet>> {
+
+	@Override
+	public int getAnswer(List<BitSet> receitas) {
+		int n = receitas.size();
+		if(n == 0) return -1;
+		
+		List<BitSet> cumulativeOr = new Vector<BitSet>(n);

```

Comment:
Usa o ArrayList que é mais rápido, só usa o Vector quando for aplicação concorrente.